### ISSUE TITLE
graphics and visual defects

### ISSUE THREAD

**Issue by Kvant9** [lightvector/KataGo#448]
Hello. @lightvector, thank you very much for creating and constantly working to improve a very good engine! I have an AMD Ryzen 3 3200G CPU with an integrated Vega 8 graphics card. It works fine with Katagomo_Lizzie_Package_20200725, but with subsequent versions of Katago, there are solid graphics and visual defects. What could be the problem (I also tried the eigen and eigenavx2 versions)?

**Comment by Friday9i** [comment]
Lightvector manages KataGo, not GUIs such as Lizzie / Sabaki or variants. I don't know this package but that should be the case too, so you should probably ask directly to the creator of the package.

**Comment by Kvant9** [comment]
That is, the whole problem can most likely be in the lizzie-yzy2.0(2.1)-shaded-win64 file.jar  (Lizzieyzy-win64.exe)?

**Comment by Kvant9** [comment]
Yes, you're right, it's all about the file: lizzie-yzy2.0(2.1)-shaded-win64.jar! So this question is not for lightvector. Please excuse me.
